roles/lighthouse/defaults/main.yaml

```diff
@@ -3,13 +3,22 @@ lighthouse_datadir: /data/lighthouse
 lighthouse_auth_jwt_path: /data/execution-auth.secret
 lighthouse_execution_engine_endpoint: http://geth:8551
 
-lighthouse_cleanup: false # when set to "true" it will remove the container
+lighthouse_cleanup: false # when set to "true" it will remove the container(s)
 
 lighthouse_ports_p2p_tcp: 9000
 lighthouse_ports_p2p_udp: 9000
 lighthouse_ports_http_beacon: 5052
 lighthouse_ports_metrics: 5054
 
+lighthouse_validator_enabled: false
+lighthouse_validator_fee_recipient: "0xF29Ff96aaEa6C9A1fBa851f74737f3c069d4f1a9" # theprotocolguild.eth
```

Comment:
why don't you just use {{ ethereum_node_cl_validator_fee_recipient }} instead? That way if we want to deploy something, we just have to change the fee_recipient in one place, instead for each client individually.

Reply:
That's exactly how it's working currently. If you are using the ethereum_node role and set the  `ethereum_node_cl_validator_fee_recipient` then it will overwrite that var on each client. You can see it being injected everywhere to the individual client roles in `ethereum_node/tasks/setup_cl.yaml`